src/homepageExperience/components/steps/Tokens.tsx

```diff
@@ -62,7 +64,11 @@ export const Tokens: FC<OwnProps> = ({
   )
 
   useEffect(() => {
-    dispatch(getAllResources())
+    try {
+      dispatch(getAllResources())
+    } catch (e) {
```

Comment:
Could you replace `catch (e)` throughout the pr with maybe `catch(err)`? `(e)` is a little ambiguous because it could mean `error` or `event`.